ddtrace/profiling/profiler.py

```diff
@@ -165,6 +172,9 @@ def _build_default_exporters(self):
             # to the agent base path.
             endpoint_path = "profiling/v1/input"
 
+        if self._lambda_function_name:
+            self.tags.update({"functionname": self._lambda_function_name.encode()})
```

Comment:
```suggestion
            self.tags.update({"functionname": self._lambda_function_name.encode("utf-8")})
```

nit: just good to be explicit imo